roadmap.md

```diff
@@ -31,48 +31,61 @@ be refined if appropriate.
 * P0. Stable API for Remote execution including platform description
 * P0. List of feature to deprecate until version 1.0 are tracked in a publicly available document
 * P1. Bazel on Windows does not need to install MSYS
-* P2. Bazel can load workspace recursively
+* P1. Migration tool: Maven project to Bazel project
 
 #### 0.7
 2017-09
 
 * P0. Skylark is fully documented; strategy for user-provided Skylark rule documentation
-* P0. Support for Android integration testing
-* P0. Support for Robolectric test for Android
+* P0. Support for Android integration testing (on Linux)
```

Comment:
We're targeting linux initially for android testing, mac os will come after. All our testing infrastructure is built for linux, so it's additional work to get it working on mac os. We can make mac os a p2 for 0.7, but more likely it will be in 0.8